README.rst

```diff
@@ -918,6 +975,69 @@ configuration status of a device changes, this ensures the check reacts
 quickly to events happening in the network and informs the user promptly
 if there's anything that is not working as intended.
 
+Iperf
+~~~~~
+
+This check provides network performance measurements such as maximum achievable bandwidth, jitter, datagram loss etc of the device using `iperf3 utility <https://iperf.fr/>`_. 
+
+It also supports tuning of various parameters related to timing, buffers and protocols (TCP, UDP with IPv4 and IPv6).
+
+This check is ``disabled`` by **default**, but you may choose to enable auto creation of this check by setting
+`OPENWISP_MONITORING_AUTO_IPERF <#OPENWISP_MONITORING_AUTO_IPERF>`_ to ``True``.
+
+You can also change the default values used for iperf checks using
+`OPENWISP_MONITORING_IPERF_CHECK_CONFIG <#OPENWISP_MONITORING_IPERF_CHECK_CONFIG>`_ setting.
+
+Instructions to configure Iperf Check
```

Comment:
I would create a heading `Usage Instructions` and move this section under the sub-heading `How to configure iperf check`.  This is make it similar to REAMDE of openwisp-controller. 

Wait for @nemesisdesign opinion before going ahead with this change.

Reply:
I am not sure I understood correctly but if you look at the result generated at https://github.com/openwisp/openwisp-monitoring/tree/issue-385/iperf-check#iperf-1, it doesn't look good (this heading is bigger than the one above), so anything better than that will be ok for me, I am pretty sure we have to give it a last round of refinement anyway @pandafy.

Reply:
Improved docs :+1: